woocommerce/woocommerce-gutenberg-products-block
# Add checkout error state

Shows an error state if a draft order cannot be created, for example, if it's out of stock.

Fixes #2060

### Screenshots

![Screenshot 2020-03-30 at 16 21 01](https://user-images.githubusercontent.com/90977/77930355-b4189600-72a2-11ea-8f45-e7bfb916bd36.png)

### How to test the changes in this Pull Request:

1. Add item to cart.
2. Edit the item and make it out of stock.
3. Go to block checkout. See error message.
